### DISCUSSION TITLE
consult-buffer: always show the parent dir of each buffer when switching buffers

### DISCUSSION THREAD

**Discussion by NightMachinery** [minad/consult#971] [Q&A]
I want to always show the parent dir of each buffer. (e.g., `parent/base.org`)

I have
```
(setq uniquify-buffer-name-style 'forward)
```
which does this for buffers with non-unique basenames, but not for all buffers.

**Comment by minad** [comment]
Maybe an alternative uniquify method would be a solution? In any case, buffer naming is not a Consult issue.
